python/dgl/nn/pytorch/conv/twirlsconv.py

```diff
@@ -1,10 +1,8 @@
+# pylint: disable=trailing-whitespace, line-too-long, missing-docstring, invalid-name, useless-super-delegation, comparison-with-itself, no-member
```

Comment:
Why disabling these warnings? You should address these warnings instead of disabling them.